include/libdnf5/module/module_item.hpp

```diff
@@ -132,11 +132,10 @@ class ModuleItem {
     // @replaces libdnf:module/ModuleItem.hpp:method:ModuleItem.getProfiles()
     std::vector<ModuleProfile> get_profiles() const;
 
-    // TODO(pkratoch): Similar implementation as in ModulePackageContainer::getDefaultProfiles().
     /// @return The default profiles.
     //
     // @replaces libdnf:module/ModuleItem.hpp:method:ModuleItem.getDefaultProfile()
-    std::vector<ModuleProfile> get_default_profiles() const;
+    std::vector<std::string> get_default_profiles() const;
```

Comment:
Hmm, this is a change to public API. But so far the method didn't have an implementation, so it couldn't get referenced anywhere, I guess it's OK, right?

Reply:
Yes, that was exactly my thinking. I am not sure if there could be any problem with it in this case...

Reply:
I believe there should not be any problem. AFAIK if someone would reference this method, the linking would fail, so there cannot be any valid consumer so far.